graph/push.go

```diff
@@ -25,6 +24,14 @@ import (
 
 var ErrV2RegistryUnavailable = errors.New("error v2 registry unavailable")
 
+type ImagePushConfig struct {
```

Comment:
@cpuguy83 may I ask why? Just because this is a struct needed only as a configuration paramenter to Push and not what we return through the api. Maybe what we write to OutStream should go to api/types